db/schema.rb

```diff
@@ -55,6 +55,16 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "notes", force: :cascade do |t|
+    t.bigint "rebate_form_id"
+    t.bigint "user_id"
```

Comment:
This sholdn't be here... do you have extra tables in your DB?
You can regen the schema file by doing this:

```RAILS_ENV=test bundle exec rake db:drop db:create db:migrate```

Reply:
You probably tested my PR for notes. It didn't get merged but you have left over tales